straxen/plugins/pulse_processing.py

```diff
@@ -384,20 +387,23 @@ def _count_pulses(records, n_channels, result):
     # Array of booleans to track whether we are currently in a lone pulse
     # in each channel
     in_lone_pulse = np.zeros(n_channels, dtype=np.bool_)
-
+    baseline_buffer = np.zeros(n_channels, dtype=np.float64)
```

Comment:
I think you could do with integers here as https://github.com/AxFoundation/strax/blob/master/strax/dtypes.py#L50 is stored in integers.

Reply:
Thanks, this is what I tried first and the final result is also stored as int16. However when computing the mean value I had to use a float64 since otherwise the results deviated in some cases from the expected value:

`np.mean(records_of_first_fragment_in_one_channel['baseline'])`